### ISSUE TITLE
Conf MARCOM & IA

### ISSUE THREAD

**Issue by andzi** [karina-klinkeviciute/pyconlt#95]
There is no other place to post thoughts regarding PyConf itself, so I'm posting here. 

Probably UX / MARCOM (marketing communications) content and IA (information architecture) issues: 
1. top menu has no previuos conf. entries (eg organised by year, there are lot samples (prototypes) - see DEFCON and similar)
2. where is content (at least - link to some archives) from the prev.conf's? - info would be helpfull offering new, more interesting topics (i.e. planing).
3. process & calendar (.ics format) would be helpfull to see main milestones & use for scheduling

**Comment by aidiss** [comment]
Yes, it is a problem.
The current website does not handle multiple conferences at once. The information from previous two years is available in separate flat pages and in the database but we cannot separate it clearly.  @karina-klinkeviciute can comment a bit more on what are the current obstacles

Also, there is a repository that contains info from the year 2016 and back, maintained by @sirex, however, the website is not running anywhere. I think it might be worth to run it and leave it for historical/marketing references.

Also, there are proposals, video recordings and more info from the year 2009, when very first PyConLT took place. The links are available in a repo mentioned above: https://github.com/sirex/pycon.lt/blob/master/content/2009.rst 

To sum up, it would be great to have historical event info organized and accessible easily. But it would require effort to make it happen.
